src/utils/README.md

```
@@ -57,7 +57,7 @@ Utility function to dump schema with list of models into OpenAPI-compatible JSON
 ## [prompts.py](prompts.py)
 Utility functions for system prompts.
 
-## [pydantic_ai.py](pydantic_ai.py)
+## [pydantic_ai_helpers.py](pydantic_ai_helpers.py)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)